pkg/utils/jwt.go

```
@@ -0,0 +1,27 @@
+/*
+Copyright 2022 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import "github.com/form3tech-oss/jwt-go"
+
+// GetTokenSubject extract the subject field from the jwt token
+func GetTokenSubject(token string) string {
+	claims := jwt.MapClaims{}
+	_, _ = jwt.ParseWithClaims(token, claims, nil)
```

Should we ignore the error here?

---

OK